src/side-navigation/side-navigation.scss

```diff
@@ -1,207 +1,137 @@
 
-    > .iui-sidenav-items-container {
+    > .iui-top,
+    > .iui-bottom {
+      flex-grow: 1;
       display: flex;
       flex-direction: column;
+    }
 
-      > .iui-sidenav-items {
-        flex-grow: 1;
-        display: flex;
-        flex-direction: column;
-      }
+    > .iui-top > .iui-button:not(:last-of-type) {
+      border-bottom: none;
+    }
+    
+    > .iui-bottom {
+      justify-content: flex-end;
 
-      > .iui-sidenav-items > .iui-sidenav-item-container,
-      > .iui-bottom-container > .iui-sidenav-item-container {
-        margin-bottom: -1px;
-        @include iui-sidenav-item-container;
+      > .iui-button {
+        border-bottom: none;
       }
     }
   }
 
   &.iui-collapsed {
-    min-width: $iui-l * 2;
-    max-width: $iui-l * 2;
+    min-width: $iui-icons-large * 3;
+    max-width: $iui-icons-large * 3;
   }
 
   &.iui-expanded,
-  > .iui-sidenav-content > .iui-sidenav-items-container {
+  > .iui-content {
     max-width: $iui-3xl * 3;
-  }
-
-  &.iui-expanded > .iui-expand-container > .iui-icon {
-    transform: rotate(180deg);
-  }
-}
 
-@mixin iui-sidenav-item-container {
-  display: flex;
-  flex-direction: row;
-  user-select: none;
-  height: $iui-l * 2;
-  cursor: pointer;
-  align-items: center;
-  transition: box-shadow $iui-speed-fast ease;
-  text-decoration: none;
-  box-sizing: border-box;
-  border-right: none;
-  border-left: none;
-
-  @include themed {
-    border-top: 1px solid t(iui-color-background-5);
-    border-bottom: 1px solid t(iui-color-background-5);
-    background-color: t(iui-color-background-1);
-  }
-
-  &:not(:last-child) {
-    border-bottom: none;
-  }
-
-  > .iui-sidenav-item-icon {
-    padding: 0 $iui-icons-default;
-    flex-shrink: 0;
-    height: $iui-icons-default;
-    width: $iui-icons-default;
-    transition: fill $iui-speed-fast ease;
-
-    @include themed {
-      fill: t(iui-icons-color);
+    > .iui-expand > .iui-icon {
+      transform: scaleX(-1);
     }
   }
 
-  > .iui-sidenav-item-caption {
-    transition: color $iui-speed-fast ease;
-    overflow-wrap: break-word;
-    white-space: nowrap;
+  .iui-button {
+    border-radius: 0;
+    border-left: none;
+    border-right: none;
     overflow: hidden;
-    text-overflow: ellipsis;
-    padding-right: $iui-m;
-
-    @include themed {
-      color: t(iui-text-color);
-    }
-  }
-
-  &.iui-disabled {
-    cursor: not-allowed;
+    justify-content: flex-start;
+    height: $iui-baseline * 5;
 
-    > .iui-sidenav-item-icon {
+    &,
+    &:hover {
       @include themed {
-        fill: t(iui-icons-color-actionable-disabled);
+        border-color: t(iui-color-background-5);
       }
     }
 
-    .iui-sidenav-item-caption {
-      @include themed {
-        color: t(iui-icons-color-actionable-disabled);
-      }
+    > .iui-icon {
+      width: $iui-icons-large;
+      height: $iui-icons-large;
+      flex-shrink: 0;
+    }
+  
+    > .iui-label {
+      margin-left: $iui-icons-large !important;
```

Comment:
No way to avoid important???

Reply:
Fixed by using `.iui-icon + .iui-label` to increase specificity (same thing we're doing in base button)